ShelterModule/Program.cs

```diff
@@ -48,9 +53,7 @@ private static void ApplyMigrations(IServiceProvider services)
     private static void ConfigureServices(IServiceCollection services, IConfiguration configuration)
     {
         services.AddDbContext<PetShareDbContext>(options =>
-                                                     options.UseSqlServer(configuration.
-                                                                              GetConnectionString(PetShareDbContext.
-                                                                                                      DbConnectionStringName)));
+            options.UseSqlServer(configuration.GetSection("PetShareDbConnectionString").Value));
```

Comment:
It would be better to still have this connection string in `ConnectionStrings` section. I think we can do that by setting the name of our config variable to `ConnectionStrings:PetShareDbConnectionString`. And if we decide not to do that, we should use `GetValue()` instead of `GetSection().Value`, since it's more sraightforward.